.github/workflows/changelog.yaml

```diff
@@ -6,9 +6,6 @@ on:
         branches:
             - master
 
-env:
-    BIGQUERY_TABLE: jarvis-dev-268314.deployment_analytics.deployment_events
```

Comment:
I don't know what this is.

Reply:
Me neither, but we don't use it anymore. It was removed from other places too